application/web/index.html

```diff
@@ -96,9 +97,21 @@ <h2 id="journal-title"></h2>
                             <button id="save-entry" class="primary">
                                 <span>Save Entry</span>
                             </button>
-                            <button class="three-dots-entry">
-                                <img src="./assets/vdots-entry-black.png" width="24" height="24">
+                            <button id = "three-dots-entry" class="three-dots">
```

Comment:
can you delete the whitespace in the id to make it like our styling?

so it should be id="three-dots-entry" instead of id = "three-dots-entry"